plugins/inputs/postgresql/postgresql.go

```diff
@@ -42,6 +40,20 @@ var sampleConfig = `
   ##
   address = "host=localhost user=postgres sslmode=disable"
 
+  ## connection configuration.
+
+  ## maxidle - specify the number of idle connections to maintain. see sql.DB for details.
```

Comment:
Either or. Just saying as it is it can confuse people. Have no current opinion on how it's addressed.